pkg/helm/controller.go

```diff
@@ -282,6 +282,9 @@ func job(chart *helmv1.HelmChart) (*batch.Job, *core.ConfigMap, *core.ConfigMap)
 			{
 				Name:  "KUBERNETES_SERVICE_PORT",
 				Value: "6443"},
+			{
```

Comment:
Will the shell script having this environment not set if it's false? Just wondering if we should always populate the variable (with true/false as necessary), or only populate it if it's true.

Reply:
yes it tolerates that in the shell script, it checks if "$BOOTSTRAP" == true which will be "" == true if it doesnt exist

Reply:
Ok - wasn't sure if we had a `set -u` in there that would complain about an unknown variable.